.github/dependabot.yml

```
@@ -12,6 +12,8 @@ updates:
     allow:
       # Only update internal dependencies for now while we evaluate this workflow.
       - dependency-name: "github.com/elastic/*"
+      - dependency-name: "go.opentelemetry.io/*"
+      - dependency-name: "github.com/open-telemetry/*"
```

Should we bit a bit more restrictive, at least for now?
```suggestion
      - dependency-name: "github.com/open-telemetry/opentelemetry-collector-contrib/*"
```

---

i have no issues with being more strict, updated first one as well